iml-manager-cli/src/display_utils.rs

```diff
@@ -186,6 +186,16 @@ impl IntoTable for Vec<SnapshotRetention> {
     }
 }
 
+impl IntoTable for Vec<StratagemReport> {
+    fn into_table(self) -> Table {
+        generate_table(
+            &["Filename", "Size", "Modify Time"],
+            self.into_iter()
+                .map(|r| vec![r.filename, r.size.to_string(), r.modify_time.to_rfc2822()]),
```

Comment:
Nit, it would be nice to display this as a human readable string (number-formatters). This file could be pretty large